### ISSUE TITLE
About the batch norm layer

### ISSUE THREAD

**Issue by wenli-vision** [yihui-he/resnet-cifar10-caffe#8]
Hi, why you set learning rate of the batch norm layer as zeros?
``layer {
  name: "first_conv_bn"
  type: "BatchNorm"
  bottom: "first_conv"
  top: "first_conv"
  param {
    lr_mult: 0
    decay_mult: 0
  }
  param {
    lr_mult: 0
    decay_mult: 0
  }
  param {
    lr_mult: 0
    decay_mult: 0
  }
}

**Comment by ethanhe42** [comment]
look at issue #2
